### DISCUSSION TITLE
Seeing myself in Epoptes now

### DISCUSSION THREAD

**Discussion by rweilert** [ltsp/ltsp#311] [Q&A]
Alkis,

This is on LTSP5, I run Epoptes as remoteapp as you know, and one of our
teachers does. I did an update, and I saw that it updated Epoptes too.

Now suddenly I can see myself under Found Clients and X-Hidden. Does the
teacher see me as well? I hope not (and cannot check it right now).
Normally one shouldn't see oneself anyway, or have you changed this for
some reason?

Regards
Rolf

**Comment by alkisg** [comment]
> I did an update, and I saw that it updated Epoptes too.

The latest Epoptes 20.01 release was back in January. Are you sure that you just now installed it?

> Normally one shouldn't see oneself anyway, or have you changed this for
some reason?

The code for that is using an X property (xprop/EPOPTES_CLIENT), as "see oneself" mostly means "same Xorg display" in this context. This part of the code hasn't been touched for some time.

**Comment by rweilert** [comment]
Yes, I am sure, no update since last year :) 

As long as I can think back - for years - back to older LTSP versions - I never could see my own terminal in Epoptes. That is why I was so astonished.

**Comment by alkisg** [comment]
@rweilert, btw, the reason I asked you to file an [issue in Epoptes](https://github.com/epoptes/epoptes/issues) was because I could point to a duplicate issue there: https://github.com/epoptes/epoptes/issues/61

But you filed it in LTSP instead... I'll close it from here, let's continue in https://github.com/epoptes/epoptes/issues/61.
